test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildProcessManager.cs

```diff
@@ -17,7 +17,7 @@ internal static class MSBuildProcessManager
             TimeSpan? timeout = null,
             MSBuildProcessKind msBuildProcessKind = MSBuildProcessKind.Dotnet)
         {
-            timeout = timeout ?? TimeSpan.FromSeconds(60);
+            timeout = timeout ?? TimeSpan.FromSeconds(120);
```

Comment:
If we're adding logging why increase the timeout? Don't we want it to fail in order to see more information?

Reply:
Because if it is going to hang the extra timeout shouldn't make a difference. If it doesn't, looking the output isn't going to tell us anything other than "restore completed" and "Build succeeded". So I thought these can go together

Reply:
Ah, infinite hangs ok. 👍 :shipit: